src/main/resources/db/migration/V001.3__CREATE_USERS_TABLE.sql

```diff
@@ -0,0 +1,9 @@
+create table users
+(
+    gender        smallint check (gender between 0 and 1),
```

Comment:
Should be varchar, also change order to follow the order of the entity properties